MODULE.bazel

```diff
@@ -19,13 +19,13 @@ use_repo(
 go_sdk = use_extension("//go:extensions.bzl", "go_sdk")
 go_sdk.download(
     name = "go_default_sdk",
-    version = "1.19.8",
+    version = "1.20.2",
 )
 use_repo(go_sdk, "go_toolchains")
 
 register_toolchains("@go_toolchains//:all")
 
-bazel_dep(name = "gazelle", version = "0.30.0")
+bazel_dep(name = "gazelle", version = "0.32.0")
```

Comment:
Do we need to raise this? In general, with Bzlmod, we should keep all deps at the lowest version possible.

Reply:
hmm, maybe not, if rules_go only needs Gazelle for testing itself. Would we mark this is as dev dependency?

Reply:
I think that we need the dep, but we don't need to update the version: The tests passed even without the bump. MVS will take care of it when used by end users.

If you just want to test against the new version, we could use `single_version_override`, which is ignored if `rules_go` isn't the root module.